.travis.yml

```diff
@@ -5,6 +5,7 @@ language: php
 php:
     - 7.1
     - 7.2
+    - 7.3
```

Comment:
I expect CI to fail for this version because of php-cs-fixer `~2.7.1` (forces <7.3).

Reply:
See fix in master to bump php-cs-fixer, once this is merged to PR it will need to be rebased, I'm planning to help @dspe on that.